updateOpenFinConfigForDeployment.js

```diff
@@ -0,0 +1,29 @@
+const semver = require('semver');
+const configBuilder = require('openfin-config-builder');
+
+if (process.argv.length <= 2) {
+    console.error(`Usage: node ${__filename} <deployName>`);
+    process.exit(1);
+}
+
+const deployName = process.argv[2];
+const deployedUrl = `http://rjmcneill.github.io/StockFlux/${deployName}/`;
+
+if ((semver.valid(deployName) !== (deployName)) && (deployName !== 'master') && (deployName !== 'dev') && (deployName !== 'chore/deployable-build')) {
```

Comment:
Could possibly make this a bit more descriptive and easier to easier to read? Maybe:

```js
const validDeployableBranchNames = ['master', 'dev', 'chore/deployable-build'];
const isInvalidDeployableBranchName = (name) => validDeployableBranchNames.indexOf(name) === -1;
const isInvalidSemver = (name) => semver.valid(name) === null;

if (isInvalidSemver(deployName) && isInvalidDeployableBranchName(deployName)) {
    // ...
```